src/exampleTypes/ResultCount.js

```diff
@@ -1,11 +1,14 @@
 import _ from 'lodash/fp'
 import { observer } from 'mobx-react'
 import { withNode, withInlineLoader } from '../utils/hoc'
+import { toNumber } from '../utils/format'
 
-let ResultCount = ({ node, display = x => x }) =>
-  node.context.response.results.length
-    ? display(node.context.response.totalRecords)
+let ResultCount = ({ node, display = x => x }) => {
```

Comment:
`display` is already a thing. You can just default display to `toNumber`

Reply:
done.